template/frameworks/hapi/server/index.js

```diff
@@ -1,24 +1,28 @@
-const Hapi = require('hapi')
 const consola = require('consola')
-const HapiNuxt = require('hapi-nuxt')
+const Hapi = require('@hapi/hapi')
+const HapiNuxt = require('@nuxtjs/hapi')
 
-const server = new Hapi.Server({
-  host: process.env.HOST || '127.0.0.1',
-  port: process.env.PORT || 3000
-})
+function start() {
```

Comment:
@clarkdo Shouldn't the function be `async`? (I'm having an error at this line on fresh installs)

Reply:
Nice catch! Will add a fix.

Reply:
Already [fixed](https://github.com/nuxt/create-nuxt-app/commit/a8ecdb93476753a2e2d0ab73504c73c03c985d1a#diff-ab6abf3f2a5af12e7f8b2e276a5c9ea2R5). waiting for release.

Reply:
@halimb @pi0 v2.8.0 has been released.